cc-snapshot

```diff
@@ -1,13 +1,42 @@
 #!/bin/bash
 
-set -eu
-set -o pipefail
-
 if [[ $EUID -ne 0 ]]; then
    echo "This script must be run as root"
    exit 1
 fi
 
+###### Update section ######
+
+#Download latest version, assign versions to variables, curlstat variable ensures update only executes if curl succeeds.
+#------------------------------------------------
+SCRIPT=$0
+curl -s -o /tmp/tmp-snap https://raw.githubusercontent.com/ChameleonCloud/cc-snapshot/fhlb_test/cc-snapshot
```

Comment:
Make sure to change the branch name back to `master` or whatever we will be using.

Reply:
Yes, thank you!